spring-data-jpa/src/test/java/org/springframework/data/jpa/repository/sample/UserRepository.java

```diff
@@ -18,15 +18,29 @@
 import jakarta.persistence.EntityManager;
 import jakarta.persistence.QueryHint;
 
-import java.util.*;
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
 import java.util.stream.Stream;
 
-import org.springframework.data.domain.*;
```

Comment:
These changes seem fine as we have less than 10 imports.

Reply:
This would be consistent with that fact I have `10` applied to non-static import statements.